### PR TITLE
Alycia

### PR DESCRIPTION
**Pull request by anclark686**
Adds two new tests: 
 - One to create a new card
 - One to add a like to an invalid card

### CODE REVIEW

File: tests/test_routes.py [hunk 1]
```diff
@@ -27,4 +29,56 @@ def test_add_like_missing_card(client):
...
+        "message": "Test card"
+    }
+
+    new_card = Card.query.get(1)
+    assert new_card
+    assert new_card.message == "Test card"
+    assert new_card.likes_count == 0
+
+
+# def test_post_task_ids_to_goal(client, one_goal, three_tasks):
```
Comment (dorl9039) [inline tests/test_routes.py]: Did you mean to leave the commented part below in?

Comment (anclark686) [reply tests/test_routes.py]: looks like this is an outdated version 
image
 this comment doesn't appear in the latest changes:

Comment (dorl9039) [reply tests/test_routes.py]: Sorry about that!

**Review by dorl9039 — APPROVED** [review]
Nice tests!

**Pull request merged**